app/main.ipynb

```diff
@@ -2,18 +2,26 @@
  "cells": [
   {
    "cell_type": "code",
-   "execution_count": null,
    "metadata": {
     "collapsed": true,
     "pycharm": {
      "name": "#%%\n"
+    },
+    "ExecuteTime": {
+     "end_time": "2024-11-13T07:57:54.724805Z",
+     "start_time": "2024-11-13T07:57:53.988866Z"
     }
    },
-   "outputs": [],
    "source": [
+    "import sqlite3\n",
     "import pandas as pd\n",
-    "import numpy as np"
-   ]
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "conn = sqlite3.connect(\"../db.sqlite3\")"
```

Comment:
btw better call it `connection` because you just shortened a little but it's less readable